packages/application-generic/src/usecases/get-subscriber-preference/get-subscriber-preference.usecase.ts

```diff
@@ -23,6 +23,7 @@ export class GetSubscriberPreference {
     private memberRepository: MemberRepository,
     private notificationTemplateRepository: NotificationTemplateRepository,
     private getSubscriberTemplatePreferenceUsecase: GetSubscriberTemplatePreference,
+    @Inject(forwardRef(() => AnalyticsService))
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.